genesyscloud/routing_sms_addresses/genesyscloud_routing_sms_addresses_schema.go

```diff
@@ -0,0 +1,100 @@
+package genesyscloud
+
+import (
+	gcloud "terraform-provider-genesyscloud/genesyscloud"
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	registrar "terraform-provider-genesyscloud/genesyscloud/resource_register"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// SetRegistrar registers all the resources, data sources and exporters in the package
+func SetRegistrar(l registrar.Registrar) {
+	l.RegisterDataSource(resourceName, DataSourceRoutingSmsAddress())
+	l.RegisterResource(resourceName, ResourceRoutingSmsAddress())
+	l.RegisterExporter(resourceName, RoutingSmsAddressExporter())
+}
+
+func RoutingSmsAddressExporter() *resourceExporter.ResourceExporter {
+	return &resourceExporter.ResourceExporter{
+		GetResourcesFunc: gcloud.GetAllWithPooledClient(getAllRoutingSmsAddress),
+	}
+}
+
+func ResourceRoutingSmsAddress() *schema.Resource {
+	return &schema.Resource{
+		Description: `Genesys Cloud routing sms address`,
+
+		CreateContext: gcloud.CreateWithPooledClient(createRoutingSmsAddress),
+		ReadContext:   gcloud.ReadWithPooledClient(readRoutingSmsAddress),
+		DeleteContext: gcloud.DeleteWithPooledClient(deleteRoutingSmsAddress),
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		SchemaVersion: 1,
+		Schema: map[string]*schema.Schema{
+			`name`: {
+				Description: `Name associated with this address`,
+				Required:    true,
+				ForceNew:    true,
```

Comment:
Should all the properties be `ForceNew`?

Reply:
Is there a specific reason why we are doing a ForceNew.  Is there something goody with the API

Reply:
@dginty4 @carnellj-genesys It's because there is no PUT or PATCH for this endpoint